modules/ossm-config-sampling.adoc

```diff
@@ -18,6 +18,13 @@ In a basic installation, `spec.tracing.sampling` is set to `10000`, which sample
 
 Setting the value to `10000` is useful for debugging, but can affect performance. For production, set `spec.tracing.sampling` to `100.`
 
+[NOTE]
+====
+The Envoy proxy sampling rate applies for applications that are available to a Service Mesh, and use the Envoy proxy. This sampling rate determines how much data the Envoy proxy collects and tracks.
+
+The Jaeger agent sampling rate applies to applications that are external to the Service Mesh, and do not use the Envoy proxy, such as a database. This sampling rate determines how much data the Jaeger agent collects and tracks. For more information, see xref:../../jaeger/jaeger_install/rhbjaeger-deploying.adoc#jaeger-config-sampling_jaeger-deploying[Configurating Jaeger sampling].
```

Comment:
There is no such thing as `Jaeger agent sampling`

There are two sampling approaches in Jaeger - directly configured in Jaeger client or Jaeger remote sampling.

Reply:
@pavolloffay and @JStickler in this case, are we discussing sampling for the Jaeger client or Jaeger remote sampling?

Would "The Jaeger agent sampling rate applies to applications that are external to the Service Mesh..."

become "The Jaeger remote sampling rate applies to applications that are external to the Service Mesh..."

Thoughts?

Reply:
> The Jaeger remote sampling rate applies to applications that are external to the Service Mesh

this looks better. Both Jaeger client sampling and Jaeger remote sampling are used in workloads outside of the service mesh

Reply:
Great -- Thanks @pavolloffay! I made the change and submitted the update.